src/components/Slide/SlideBase/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import { Slide as MaterialSlide, SlideProps as MaterialSlideProps } from "@material-ui/core";
+
+export interface SlideBaseProps extends MaterialSlideProps {}
+
+const SlideBase = React.forwardRef((props: SlideBaseProps, ref) => {
```

Comment:
Please add generic here

Reply:
![image](https://user-images.githubusercontent.com/58205909/131253154-f7cfd4f1-e77a-4090-94bb-daf53f6fdc57.png)

Reply:
![image](https://user-images.githubusercontent.com/58205909/131279785-43f31ac0-cec4-4a10-a678-856c3b35ac9d.png)
I choose generic for Ref is unknown because it is set Ref<unknown> in the root source.